main/admin/course_edit.php

```diff
@@ -283,70 +306,54 @@
     // Check if the visual code is already used by *another* course
     $visual_code_is_used = false;
 
-    $warn = get_lang('TheFollowingCoursesAlreadyUseThisVisualCode').':';
+    $warn = get_lang('TheFollowingCoursesAlreadyUseThisVisualCode') . ':';
```

Comment:
The column (:) at the could be removed. It should be part of the string or ignored completely (otherwise may cause issues with RTL languages)